srcpkgs/cjs/template

```diff
@@ -1,27 +1,29 @@
 # Template file for 'cjs'
 pkgname=cjs
-version=4.6.0
+version=4.8.0
 revision=1
-build_style=gnu-configure
+build_style=meson
 build_helper=gir
-configure_args="--disable-static --disable-profiler"
-hostmakedepends="automake autoconf-archive libtool gnome-common pkg-config
- gobject-introspection glib-devel dbus mozjs52-devel"
+configure_args="-Dskip_dbus_tests=true -Dskip_gtk_tests=true
+ -Dprofiler=disabled -Dinstalled_tests=false" # Disable tests because they need dbus, X and GTK, same as gjs.
```

Comment:
```suggestion
# Disable tests because they need dbus, X and GTK, same as gjs.
configure_args="-Dskip_dbus_tests=true -Dskip_gtk_tests=true
 -Dprofiler=disabled -Dinstalled_tests=false"
```

That said, for tests that depend on dbus you should use `checkdepends="dbus"` and run them with `dbus-run-session ninja check`. For X, you can add `xvfb-run` into the mix.

Reply:
Yes, it should be possible, but I am not going to do that.